src/utils/filter.tsx

```diff
@@ -0,0 +1,35 @@
+import { getTodos } from '../api';
+import { Todo } from '../types/Todo';
+
+export enum FilterType {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
+
+export async function filterTodos(
+  filterType: FilterType,
+  query: string,
+) {
+  let todosFromServer = await getTodos();
```

Comment:
You should not to make API call here, you should get todos as an argument of function